eland/dataframe.py

```diff
@@ -808,7 +807,12 @@ def _index_summary(self):
         return f"{name}: {len(self)} entries{index_summary}"
 
     def info(
-        self, verbose=None, buf=None, max_cols=None, memory_usage=None, null_counts=None
+        self,
+        verbose: Optional[bool] = None,
+        buf: Optional[StringIO] = None,
+        max_cols: Optional[int] = None,
+        memory_usage: Optional[bool] = None,
+        show_counts: Optional[bool] = None,
```

Comment:
The change from `null_counts` to `show_counts` isn't explained anywhere?

Reply:
To keep inline with pandas => https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.info.html

They have deprecated `null_counts` and replaced it with `show_counts`